_posts/2023-04-14-h2oai.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  "The Return of the H2oai Benchmark"
+author: Tom Ebergen
+excerpt_separator: <!--more-->
+---
+
+*TL;DR: We re-ran the H2O.ai db-benchmarks with up-to-date libraries and DuckDB is consistently a top performer.*
```

Comment:
maybe less on the nose: "We've resurrected the H2O.ai db-benchmark with up to date libraries and plan to keep re-running it."